setup.py

```diff
@@ -190,6 +191,11 @@ def run(self):
             elif os.path.isdir(f):
                 shutil.rmtree(f)
 
+
+        for name in glob.glob(dir_setup+"/doc/src/modules/physics/vector/*.pdf"):
+            if os.path.isfile(name):
+                os.remove(name)
```

Comment:
Why do you think glob only works on UNIX based systems? The docs don't say there are any limitations: https://docs.python.org/2/library/glob.html. Where is this limitation discussed?